internal/collectors/github/organization_collector.go

```diff
@@ -126,3 +142,25 @@ func (c *organizationCollector) collectOrgSamlData(org string) (*bool, error) {
 	return &samlEnabled, nil
 
 }
+
+func (c *organizationCollector) collectEnterpriseVisibilityChangePolicyDisabled() (*bool, error) {
+	enterpriseName, enterpriseExist := context_utils.GetEnterprise(c.Context)
+	visibilityChangeEnabled := false
+	if enterpriseExist {
+		if len(enterpriseName) == 0 {
+			return &visibilityChangeEnabled, errors.New("cannot fetch info for empty enterprise")
+		}
+	}
+	variables := map[string]interface{}{
+		"slug": githubv4.String(enterpriseName),
+	}
+
+	err := c.Client.GraphQLClient().Query(c.Context, &enterpriseVisibilityChangePolicyQuery, variables)
+
+	if err != nil {
+		return nil, err
+	}
+	visibilityChangeEnabled = enterpriseVisibilityChangePolicyQuery.Enterprise.OwnerInfo.MembersCanChangeRepositoryVisibilitySetting == "DISABLED"
```

Comment:
what happens when it is not touched at all on the enterprise level?

Reply:
We should treat untouched policy as if it's ENABLED